assets/jsx/uploader.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import filesize from 'filesize'
+
+class Uploader extends React.Component {
```

Comment:
See previous comment re:class names – I'd vote for something like `UploadStatus`